<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>820</width>
    <height>526</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Binaitic</string>
  </property>
  <property name="styleSheet">
   <string notr="true">MainWindow{
	background-color : #9980FA;
}</string>
  </property>
  <widget class="QWidget" name="Binaitic">
   <widget class="QPushButton" name="pb_Quitter">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>430</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>&amp;Quitter</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Title">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>20</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>BINAITIC</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Mutation">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>200</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Mutation rate :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Equation">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>70</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Equation</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Generations">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>170</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Générations :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Alpha">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>260</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Alpha : </string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_Lancer">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>360</y>
      <width>171</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>&amp;Lancer</string>
    </property>
   </widget>
   <widget class="MonCanvas" name="Canvas_Figure" native="true">
    <property name="geometry">
     <rect>
      <x>109</x>
      <y>79</y>
      <width>521</width>
      <height>251</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Equation">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>90</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_LessFitness">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>240</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Less fitness :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Population">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>140</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Population :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Population_res">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>140</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_Generations_res">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>170</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_Mutation_res">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>200</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_LessFitness_res">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>240</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_Precedent">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>360</y>
      <width>171</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>&amp;Précédent</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_Suivant">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>360</y>
      <width>171</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>&amp;Suivant</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Crossover">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>220</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Crossover rate  :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Crossover_res">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>220</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_Generation_num">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>60</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Generation n° :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Generation_num_res">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>60</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_Alpha_res">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>260</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_X_axis_max_res">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>330</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_X_axis_min_res">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>330</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_Y_axis_min_res">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>310</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_Y_axis_max_res">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>80</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>820</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MonCanvas</class>
   <extends>QWidget</extends>
   <header location="global">moncanvas.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
